dags/nocp/scripts/run_ocm_benchmark.sh

```diff
@@ -0,0 +1,80 @@
+    cd /tmp/${UUID}
+    echo "Building the binary"
+    go mod download
+    make
+
+    cat /tmp/environment_new.txt | awk -v x="'" -F "=" '{print "export " \$1"="x\$2x}' > vars.sh
+    source vars.sh
+    echo "cat /tmp/environment_new.txt"
+    cat /tmp/environment_new.txt
+    rm -rf /tmp/environment_new.txt vars.sh
+
+    echo "Creating aws key with admin user for OCM testing"
+    aws_key=\$(/usr/bin/aws iam list-access-keys --user-name OsdCcsAdmin --output text --query 'AccessKeyMetadata[1].AccessKeyId')
+    if [[ \$aws_key != None ]]; then
+        /usr/bin/aws iam delete-access-key --user-name OsdCcsAdmin --access-key-id \$aws_key
+    fi
+    admin_key=\$(/usr/bin/aws iam create-access-key --user-name OsdCcsAdmin --output json)
+    export AWS_ACCESS_KEY_ID=\$(echo \$admin_key | jq -r '.AccessKey.AccessKeyId')
+    export AWS_SECRET_ACCESS_KEY=\$(echo \$admin_key | jq -r '.AccessKey.SecretAccessKey')
+    sleep 60
+
+    echo "Preparing config files"
+    envsubst < /tmp/${UUID}/ci/templates/config.yaml > /tmp/${UUID}/config.yaml
+    export KUBE_ES_INDEX=ocm-uhc-acct-mngr
+    envsubst < /tmp/${UUID}/ci/templates/kube-burner-config.yaml > /tmp/${UUID}/kube-burner-am-config.yaml
+    export KUBE_ES_INDEX=ocm-uhc-clusters-service
+    envsubst < /tmp/${UUID}/ci/templates/kube-burner-config.yaml > /tmp/${UUID}/kube-burner-cs-config.yaml
+
+    start_time=\$(date +%s)
+    echo "Running the ocm-load-test test"
+    build/ocm-load-test --config-file config.yaml &> ${UUID}_ocm-load-test
+    cp ${UUID}_ocm-load-test /tmp/${UUID}/results/${UUID}_ocm-load-test.json
+
+    end_time=\$(date +%s)
+
+    echo "Uploading Result files..."
+    python3 automation.py upload --dir /tmp/${UUID}/results --server ${SNAPPY_DATA_SERVER_URL} --user ${SNAPPY_DATA_SERVER_USERNAME} --password ${SNAPPY_DATA_SERVER_PASSWORD};
+
+    curl -LsS ${KUBE_BURNER_RELEASE_URL} | tar xz
+    echo "Running kube-burner index to scrap metrics from UHC account manager service from \${start_time} to \${end_time} and push to ES"
+    ./kube-burner index -c kube-burner-am-config.yaml --uuid=${UUID} -u=${PROM_URL} --job-name ocm-api-load --token=${PROM_TOKEN} -m=ci/templates/metrics_acct_mgmr.yaml --start \$start_time --end \$end_time
+    echo "UHC account manager Metrics stored at elasticsearch server $ES_SERVER on index $KUBE_ES_INDEX with UUID $UUID and jobName: ocm-api-load"
+
+    echo "Running kube-burner index to scrap metrics from UHC clusters service from \${start_time} to \${end_time} and push to ES"
+    export KUBE_ES_INDEX=ocm-uhc-clusters-service
+    ./kube-burner index -c kube-burner-cs-config.yaml --uuid=${UUID} -u=${PROM_URL} --job-name ocm-api-load --token=${PROM_TOKEN} -m=ci/templates/metrics_clusters_service.yaml --start \$start_time --end \$end_time
+    echo "UHC clusters service metrics stored at elasticsearch server $ES_SERVER on index $KUBE_ES_INDEX with UUID $UUID and jobName: ocm-api-load"
+
+EOF
```

Comment:
Is there any reason we don't clean up the working directory when we are done?

Reply:
I thought this will allow us login to jumphost and debug if we hit any failures. May be once this CI becomes mature, we can add the cleanup.

Reply:
Can we cleanup the keys/creds that are generated? I'm fine w/ leaving the rest for now.

Reply:
sure. I will update the PR with that.

Reply:
Cleanup keys/creds added now i.e

    echo "Removing configuration files"
    rm -f /tmp/${UUID}/config.yaml
    rm -f /tmp/${UUID}/kube-burner-am-config.yaml
    rm -f /tmp/${UUID}/kube-burner-cs-config.yaml